src/storage/storage.bif

```diff
@@ -0,0 +1,24 @@
+##! Functions related to general storage operations. These are not specific to async or sync.
+
+%%{
+
+#include "zeek/storage/Backend.h"
+
+using namespace zeek;
+using namespace zeek::storage;
+
+%%}
+
+module Storage;
+
+## Checks whether a storage backend is connected.
+##
+## backend: A handle to the backed to check.
+##
+## Returns: T if the backend is open, F if the backend is not connected or if the handle
+##          is invalid.
+function Storage::is_connected%(backend: opaque of Storage::BackendHandle%) : bool
```

Comment:
Would you have gripes renaming this to `Storage::is_open` ? The accessor on `Backend` is also `IsOpen()`. The consistency would help reduce cognitive overhead and "connection" (to me anyway) sounds like a remote connection, which isn't necessarily true.

Reply:
Seems reasonable to me.